### DISCUSSION TITLE
Deployment failing for production

### DISCUSSION THREAD

**Discussion by Virous77** [vercel/community#5518] [Help] [answered]
### Today i was working on my personal web apps and updated some packages and some other things and when push the code its failed on production.

Got this error :
![Image 18-01-24 at 11 14 PM](https://github.com/vercel/community/assets/101452588/8dd3a33f-fed1-4292-9e8b-145c556031a2)

### While I build locally and its working too fast and well as its always do.

![Image 18-01-24 at 11 19 PM](https://github.com/vercel/community/assets/101452588/c7791e81-69b3-4f42-86f9-e85a1aece622)

**Comment by swarnava** [comment, marked as answer]
Hi,

You can upgrade to Next.js 14.1 which should resolve this issue.

**Reply by Virous77** [reply]
Cool Thank you.

I was able to fix that issue after downgrading those updated packages. But will try again after latest package and next with 14.1.0.
